cloud_experts_tutorials/cloud-experts-entra-id-idp.adoc

```diff
@@ -157,41 +191,59 @@ $ rosa create idp \
 ----
 
 [id="rosa-mobb-azure-oidc-grant-permissions"]
-== Grant additional permissions to individual users
+== Granting additional permissions to individual users and groups
+
+After a few minutes, the cluster authentication Operator reconciles your changes, and you can log in to the cluster by using Entra ID.
```

Comment:
Should this sentence go before the "Granting additional permissions..." heading? It seems like it is the result of the previous procedure.